pkg/power/platform/source/redfish_util.go

```diff
@@ -67,7 +69,10 @@ func getRedfishModel(access RedfishAccessInfo, endpoint string, model interface{
 	if err != nil {
 		return err
 	}
-	defer resp.Body.Close()
+	defer func() {
+		io.Copy(ioutil.Discard, resp.Body)
```

Comment:
can you fix the lint error
```
Error return value of `io.Copy` is not checked (errcheck)
```

Reply:
Fixed, all checkes seems to be succeeded. Please review again.